src/keyboard.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "game_object.h"
+#include "keyboard_listener.h"
+#include "observable.h"
+#include <SDL2/SDL.h>
+#include <string>
+#include "graphics_handler.h"
+#include "events_listener.h"
```

Comment:
Let's organize our includes to be:

1. System libraries
2. Third party libraries
3. Project libraries